src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/AnalyzerConfigItemHandler.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+
+using Microsoft.VisualStudio.ProjectSystem.Logging;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices.Handlers
+{
+    /// <summary>
+    ///     Handles changes to the &lt;EditorConfigFiles/&gt; items during design-time builds.
```

Comment:
Is the item actually named "EditorConfigFiles", if so, it should be "EditorConfigFile" -> items are not plural.

Reply:
It is in fact EditorConfigFiles.

Reply:
Can we fix that? Items are not plural.

Reply:
Maybe--we'll need to get the compiler team to buy off on that. I can't say it's at the top of my priority list at the moment, though. :-)